crates/typst-ide/src/tests.rs

```diff
@@ -119,6 +119,7 @@ impl IdeWorld for TestWorld {
     fn packages(&self) -> &[(PackageSpec, Option<EcoString>)] {
         const LIST: &[(PackageSpec, Option<EcoString>)] = &[(
             PackageSpec {
+                // NOTE: DEFAULT_NAMESPACE (i.e. "preview") in typst-kit/src/package.rs
                 namespace: EcoString::inline("preview"),
```

Comment:
Ideally we'd just use the constant here. Maybe it can be moved to `typst-utils` if it's shared by multiple Typst crates, but dunno (cc @laurmaedje)

Reply:
My reasoning: the test exists to test it. 😇 

In other words, imagine we decide to promote "preview" to "public". But when we change the const we name it "puvlic", with a typo. 

In this case, if the test used the const directly, the test passes. If not, the test fails (unless we make the same typo twice). 

Let me know what you think. Glad to update it as you like, just wanted to share my thought process first.

Reply:
Ah, I see. This seems like robust reasoning to me.

Maybe we could add a sentence to the comment saying "Keep as a literal so the test fails if it changes."

Reply:
I updated the comment to reflect this thinking.